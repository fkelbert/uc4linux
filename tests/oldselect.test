#!/bin/sh

# Check oldselect syscall decoding.

. "${srcdir=.}/init.sh"

run_prog
run_strace -e oldselect $args
match_grep

exit 0
