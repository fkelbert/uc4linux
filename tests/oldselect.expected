oldselect\(2, \[1\], \[0\], NULL, \{0, 42\}\) += 0 \(Timeout\)
